projects/packages/waf/src/class-waf-rules-manager.php

```diff
@@ -223,15 +222,15 @@ public static function generate_rules() {
 		Waf_Runner::initialize_filesystem();
 
 		$rules                = "<?php\n";
-		$entrypoint_file_path = Waf_Runner::get_waf_file_path( self::RULES_ENTRYPOINT_FILE );
+		$entrypoint_file_path = Waf_Runner::get_waf_file_path( Waf_Runner::ENTRYPOINT_FILE );
```

Comment:
I am wondering if this is problematic - will this `generate_rules` method run within a request where an outdated `Waf_Runner` class was autoloaded in standalone mode, creating the same issue?

Wondering if we just need to duplicate the deceleration of the entrypoint file value, or provide it from a third source separate from the runner and rule manager classes.

🤔